src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -1188,7 +1175,7 @@ public double GetNetworkWeight()
         }
 
         /// <inheritdoc/>
-        public Models.GetStakingInfoModel GetGetStakingInfoModel()
+        public Models.GetStakingInfoModel GetStakingInfoModel()
```

Comment:
Before it was called `GetGetStakingInfoModel` for the consistency. So the pattern for rpc is : `get` + `command name` + `model`. The command here is `getstakinginfo` so that's where this double 'get' came from.